src/views/Drawer/DrawerNavigatorItems.js

```diff
@@ -26,8 +26,9 @@ type Props = {
   getLabel: (scene: DrawerScene) => ?(React.Element<*> | string),
   renderIcon: (scene: DrawerScene) => ?React.Element<*>,
   onItemPress: (info: DrawerItem) => void,
-  style?: ViewStyleProp,
-  labelStyle?: TextStyleProp,
+  itemsContainerStyle?: ViewStyleProp,
+  itemStyle?: ViewStyleProp,
+  labelStyle?: TextStyleProp
```

Comment:
I think you need to add a ',' at the end of `labelStyle?: TextStyleProp` to fix the prettier issue